examples/tutorials/03_ready_to_use_model_tf.ipynb

```diff
@@ -69,7 +69,7 @@
      "name": "stderr",
```

Comment:
<p>Many of notebooks has been updated since you started this PR: for example, <code>default</code> parameter is no longer needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>You don't need <code>before</code> at all here in the first place</p>